api-report/azure-client.api.md

```diff
@@ -89,9 +96,21 @@ export interface AzureMember<T = any> extends IMember {
     userName: string;
 }
 
+// @public
+export interface AzureRemoteConnectionConfig extends AzureConnectionConfig {
+    tenantId: string;
+    type: "remote";
+}
+
 // @public
 export type IAzureAudience = IServiceAudience<AzureMember>;
 
+// @public
+export function isAzureLocalConnectionConfig(connectionConfig: AzureConnectionConfig): connectionConfig is AzureLocalConnectionConfig;
+
+// @public
+export function isAzureRemoteConnectionConfig(connectionConfig: AzureConnectionConfig): connectionConfig is AzureRemoteConnectionConfig;
+
```

Comment:
These don't need exported. I'm guessing that the index.ts file just * exports interfaces. I would recommend moving these into a utils.ts file or doing select export from the interfaces.ts file

Reply:
done 👍